main.tf

```diff
@@ -1,52 +1,52 @@
 # Define composite variables for resources
 module "label" {
   source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.3.5"
```

Comment:
update to https://github.com/cloudposse/terraform-null-label/releases/tag/0.12.0, it's for TF 0.12